packages/config/src/tokens/generated.json

```diff
@@ -4096,6 +4246,63 @@
         "slug": "omnichain"
       }
     },
+    {
+      "id": "arbitrum:rseth-kelpdao-restaked-eth",
+      "name": "KelpDao Restaked ETH",
+      "coingeckoId": "kelp-dao-restaked-eth",
+      "address": "0x4186BFC76E2E237523CBC30FD220FE055156b41F",
+      "symbol": "rsETH",
+      "decimals": 18,
+      "deploymentTimestamp": 1707187495,
+      "coingeckoListingTimestamp": 1705017600,
+      "category": "other",
+      "iconUrl": "https://assets.coingecko.com/coins/images/33800/large/Icon___Dark.png?1702991855",
+      "chainId": 42161,
+      "type": "EBV",
```

Comment:
should be NMV, it's an omnichain token

example: 
- outbound tx: https://blastexplorer.io/tx/0x9e281686263cb43082c6626ec21e19573b64964337906ae1a95f4a9aba13cb19
- inbound tx: https://arbiscan.io/tx/0xe08ad97dd3e091ef4f3cc43a387d04cb2678775918ef693d4bdba653029d4487